docs/Field-Mapping.md

```diff
@@ -243,7 +244,7 @@ All payment methods:
     * `iban` -> `IBAN` of type String (CT initial, but overridden by PAYONE)
     * `bic`  -> custom `BIC` (CT initial, but overridden by PAYONE)
   * traditional identification:
-    * `bankcountry` -> `bankCountry`
+    * `bankcountry` -> custom field `bankCountry` of Type String on the CT Payment (required for bancontact and iDeal) 
```

Comment:
can you also add `bankgrouptype` field which is only used for iDeal?

Reply:
https://github.com/commercetools/commercetools-payone-integration/pull/313/commits/c32f4740082c9710b93abd26918c87d3e26d6eda